cmd/contour/serve.go

```diff
@@ -205,6 +208,14 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 	informers = registerEventHandler(informers, dynamicInformers.ForResource(projectcontour.TLSCertificateDelegationGVR).Informer(), eh)
 	informers = registerEventHandler(informers, coreInformers.Core().V1().Services().Informer(), eh)
 
+	if ctx.UseServiceAPITypes {
+		log.Info("Enabling EXPERIMETNAL SERVICE API TYPES")
```

Comment:
IF YOU'RE GOING TO YELL, PLEASE SPELL EXPERIMENTAL CORRECTLY :)

But seriously, this isn't work yelling about, no need for the shout caps.

Reply:
Sure, it was easier to pick it out in caps, but I'll change it.